### PR TITLE
add mrfioc2 and devlib2 support

### PR DESCRIPTION
**Pull request by agaget**
Using of the standard driver, customs modifications from CEA will be included at one point one day. It is not necessary for standard using.

In draft, because it needs to be tested with hardware.

### CODE REVIEW

**Comment by agaget** [comment]
Not a draft, anymore. 2.6.0 (with my mask features) fully tested in laboratory

**Comment by agaget** [comment]
can it be reviewed ?

**Comment by agaget** [comment]
> can it be reviewed please ?

or no ?

**Review by stephane-cea — COMMENTED** [review]
It looks good to me!

But I think I'm not the main reviewer here, I'll let @minijackson approve and merge.

File: pkgs/epnix/support/devlib2/default.nix
Comment (minijackson) [inline pkgs/epnix/support/devlib2/default.nix:28]: devlib2 seems to be developed under the EPICS license: https://github.com/epics-modules/devlib2/blob/master/LICENSE

File: pkgs/epnix/support/devlib2/default.nix
Comment (minijackson) [inline pkgs/epnix/support/devlib2/default.nix:14]: If tests don't pass, can you add a comment explaining what happens?

File: pkgs/epnix/support/mrfioc2/default.nix
Comment (minijackson) [inline pkgs/epnix/support/mrfioc2/default.nix:15]: If tests don't pass, can you add a comment explaining what happens?

File: pkgs/epnix/support/mrfioc2/default.nix
Comment (minijackson) [inline pkgs/epnix/support/mrfioc2/default.nix:33]: `meta` should be the last attribute

File: pkgs/epnix/support/mrfioc2/default.nix
Comment (minijackson) [inline pkgs/epnix/support/mrfioc2/default.nix:40]:
```suggestion
  postInstall = ''
    if [[ -d iocBoot ]]; then
      cp -rafv iocBoot -t "$out"
    fi
  '';
```

File: lib/maintainers/maintainer-list.nix
Comment (minijackson) [inline lib/maintainers/maintainer-list.nix:65]: Can you put yourself at the top? The comment above says to keep the list alphabetically sorted

**Review by minijackson — COMMENTED** [review]
Thanks! Only minor comments. Could you split these changes into 2 commits? One for devlib2 / adding yourself as maintainer, and the other one for mrfioc2?

File: lib/maintainers/maintainer-list.nix
Comment (agaget) [reply lib/maintainers/maintainer-list.nix:65]: I'm always at the top 👓

**Comment by agaget** [comment]
I think, I have resolved everything

**Review by minijackson — APPROVED** [review]
Thanks!

**Pull request merged**
